server/lib/pbench/indexer.py

```diff
@@ -3708,9 +4211,9 @@ def mk_sosreports(self):
         self.idxctx.logger.debug("start")
 
         sosreports = [
-            x
-            for x in self.tb.getnames()
-            if x.find("sosreport") >= 0 and x.endswith(".md5")
+            x.name
```

Comment:
ditto

Reply:
Not sure how `pathlib` will help with `.find("sosreport")` here, or with `.endswith(".md5")`.

Reply:
Let's track this in issue #1584 and make this a deliberate change in a separate commit.